conf/are.conf.part

```diff
@@ -4,7 +4,7 @@
 
 source src_ch_are_landschaftstypen : def_queryable_features
 {
-    sql_db = are
+    sql_db = are_dev
```

Comment:
yes, that is how it was configured until now, so
``pgcluster.dev.bgdi.ch -> are == pg.bgdi.ch -> are_dev``